volatility3/framework/plugins/windows/pslist.py

```diff
@@ -136,6 +136,29 @@ def create_pid_filter(
                 filter_func = lambda x: x.UniqueProcessId not in filter_list
         return filter_func
 
+    @classmethod
+    def create_active_process_filter(
```

Comment:
Happy with this, adding new filters is the right way of doing things, this should be useful!  5:)